src/invision/components/movie-popup/movie.popup.js

```diff
@@ -4,14 +4,7 @@ import Modal from 'react-awesome-modal';
 export default class MoviePopup extends React.Component {
   constructor(props) {
     super(props)
-    this.state = {
-      title: this.props.movie?.title || '',
-      releaseDate: new Date(this.props.movie?.releaseDate),
-      url: this.props.movie?.url || '',
-      genre: this.props.movie?.genre || '',
-      overview: this.props.movie?.overview || '',
-      runtime: this.props.movie?.runtime || ''
-    }
+    this.state = { ...this.props.movie }
```

Comment:
It is not a good practice put to the state full object, put only needed properties.